soc/v2/gr716a/Kconfig.soc

```
@@ -0,0 +1,8 @@
+# Copyright (c) 2019-2020 Cobham Gaisler AB
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_GR716A
+	bool
+
+config SOC
+        default "gr716a" if SOC_GR716A
```

should be a tab here:
```suggestion
	default "gr716a" if SOC_GR716A
```

---

Fixed